.github/workflows/static.yml

```diff
@@ -0,0 +1,43 @@
+# Simple workflow for deploying static content to GitHub Pages
```

Comment:
I think this was from a concept you were working. This commit shouldn't be in this branch.